pybossa/forms/dynamic_forms.py

```diff
@@ -63,7 +63,7 @@ def set_product_subproduct_choices(form, config_products):
     config_products = config_products or {}
     products = list(config_products.keys())
     choices = [("", "")]
-    form.product.choices = choices + [(p, p) for p in products]
+    form.product.choices = sorted(choices + [(p, p) for p in products])
```

Comment:
If you are passing the list already sorted [here](https://github.com/bloomberg/pybossa/pull/390/files#diff-8fa3bf883c1a925427ca6a3b4587f452R764), why Is this need it?

Reply:
The first sorts the list of subproducts. The line above sorts the list of products. Both calls were actually required to sort the two drop downs. :)